.cirrus.yml

```diff
@@ -0,0 +1,10 @@
+container:
+  image: node:latest
+
+test_task:
+  node_modules_cache:
+    folder: node_modules
+    compile_script: tsc
```

Comment:
`compile_script` should be one level up (outside `node_modules_cache`)

Reply:
Thanks, that makes sense and works!  I was wondering why it never actually ran on Cirrus.

Also Best Product Support *EVER* :-) !